gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/task/AbstractJKubeTask.java

```diff
@@ -220,6 +221,8 @@ protected File getManifest(KubernetesClient kc) {
     return manifest;
   }
 
+  protected void postProcessResolvedImages() { }
```

Comment:
we'll need to discuss this again, the idea we discussed was not to post-process the resulting images, but to make sure that the configuration was stable (and hopefully immutable) before propagating it downstream.

Reply:
Actually, I had added this before the `ConfigHelper.initImageConfiguration` but I noticed that generators are invoked inside this call and resolveImages get populated after this invocation (for zero configuration scenarios).